tests/test_array.py

```diff
@@ -454,29 +470,47 @@ def test_array_functions(self, x):
         self.assertTrue(_equal_arrays(x.mean().collect(), mean))
         self.assertTrue(_equal_arrays(x.sum().collect(), sum))
 
-    @parameterized.expand([(ds.random_array((20, 30), (5, 6)),
-                            ds.random_array((30, 10), (6, 2))),
+    @parameterized.expand([(np.full((10, 10), 3, complex),),
+                           (sp.csr_matrix(np.full((10, 10), 5, complex)),)])
```

Comment:
Can we use a matrix with complex numbers like `np.random.rand(10,10)+1j*np.random.rand(10,10)`?

Reply:
Done